ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -390,17 +390,6 @@ ORK_EXTERN ORKTrailMakingTypeIdentifier const ORKTrailMakingTypeIdentifierB;
                                              options:(ORKPredefinedTaskOption)options;
 
 /**
- @Deprecated
- */
-+ (ORKOrderedTask *)audioTaskWithIdentifier:(NSString *)identifier
-                     intendedUseDescription:(nullable NSString *)intendedUseDescription
-                          speechInstruction:(nullable NSString *)speechInstruction
-                     shortSpeechInstruction:(nullable NSString *)shortSpeechInstruction
-                                   duration:(NSTimeInterval)duration
-                          recordingSettings:(nullable NSDictionary *)recordingSettings
-                                    options:(ORKPredefinedTaskOption)options __deprecated;
-
-/**
```

Comment:
This deprecation warning is removed in https://github.com/ResearchKit/ResearchKit/pull/959 and will cause a merge conflict with that branch. (It's already in stable 1.4 and 1.5 will be released soon.)